client/client.go

```diff
@@ -48,6 +48,23 @@ var DefaultTransport CancelableTransport = &http.Transport{
 	TLSHandshakeTimeout: 10 * time.Second,
 }
 
+type EndpointSelectionMode int
+
+const (
+	// EndpointSelectionDefault: pick an endpoint in a random manner
```

Comment:
Please make comments full sentences as `EndpointSelectionDefault is to pick an endpoint in a random manner.`